### PR TITLE
VB-2336 unit tests session templates

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
## Description
Added tests for data, service and route levels for the single session template page

### CODE REVIEW

File: server/routes/prisons/prisons.test.ts [hunk 1]
```diff
@@ -250,6 +250,39 @@ describe('GET /prisons/{:prisonId}/session-templates', () => {
...
+        expect($('h1').text().replace(/\s+/g, ' ')).toContain(singleSessionTemplate.name)
+
+        expect($('[data-test="reference"]').text()).toBe(singleSessionTemplate.reference)
+        expect($('[data-test="dayOfWeek"]').text()).toBe('Wednesday')
+        expect($('[data-test="startTime"]').text()).toBe(singleSessionTemplate.startTime)
+        expect($('[data-test="endTime"]').text()).toBe(singleSessionTemplate.endTime)
+        expect($('[data-test="openCapacity"]').text()).toBe(singleSessionTemplate.openCapacity.toString())
+        expect($('[data-test="closedCapacity"]').text()).toBe(singleSessionTemplate.closedCapacity.toString())
+        expect($('[data-test="validFromDate"]').text()).toBe('21 March 2023')
+        // expect($('[data-test="validToDate"]').text()).toBe('No end date')
```
Comment (tpmcgowan) [inline server/routes/prisons/prisons.test.ts:276]: Comment in other file as to why this isn't working...

**Review by tpmcgowan — COMMENTED** [review]
Looks good - just added a few comments

File: server/views/pages/prisons/viewSingleSessionTemplate.njk [hunk 2]
```diff
@@ -73,103 +73,103 @@
               text: "Reference"
             },
             value: {
-              text: sessionTemplate.reference
+              html: "<span data-test='reference'>" + sessionTemplate.reference + "</span>"
```
[the remark this answers is not in the document]
Comment (hutcheonb-moj) [reply server/views/pages/prisons/viewSingleSessionTemplate.njk]: Changed to use the suggested method, route level tests now all working correctly

**Review by tpmcgowan — APPROVED** [review]
LGTM! (minor comment about unnecessary whitespace clearing but will merge anyway to get latest code in 👍 )

**Pull request merged**